This is in fact needed on macOS, where the build otherwise fails
with the same error.

From 58a8b4e39385d5e8ef8dfd12c1f5237177340e10 Mon Sep 17 00:00:00 2001
From: Jun Wu <quark@meta.com>
Date: Tue, 18 Jun 2024 15:23:48 -0700
Subject: [PATCH] rust: fix compile on Windows

  error: field `unilateral` is never read
     --> src\lib.rs:589:17
      |
  588 |         pub struct Unilateral {
      |                    ---------- field in this struct
  589 |             pub unilateral: bool,
      |                 ^^^^^^^^^^
      |
      = note: `Unilateral` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis

diff --git a/watchman/rust/watchman_client/src/lib.rs b/watchman/rust/watchman_client/src/lib.rs
index 4b4f0b04f98e..1d102fe0b2bd 100644
--- a/watchman/rust/watchman_client/src/lib.rs
+++ b/watchman/rust/watchman_client/src/lib.rs
@@ -586,6 +586,7 @@ impl ClientTask {
         use serde::Deserialize;
         #[derive(Deserialize, Debug)]
         pub struct Unilateral {
+            #[allow(unused)]
             pub unilateral: bool,
             pub subscription: String,
             #[serde(default)]
